public/scripts/app.js

```diff
@@ -2,6 +2,94 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+// Profile GET Statement
+$.ajax({
+  method: "GET",
+  url:"/api/profile"
+})
+// DB GET Statement
+$.ajax({
+  method: "GET",
+  url: "/api/heroes",
+  success: displayHero
+})
 
+
+// Function to show seeded DB
+function displayHero(heroes){
+  heroes.forEach(function(hero){
+    renderHero(hero);
+  });
+}
+// Post function
+  $("form").on("submit", function(e){
+    e.preventDefault();
+      let userHero = $(this).serialize();
+      console.log(userHero);
+      $.post("/api/heroes", userHero, function(data){
+        renderHero(data);
+      });
+      $(this).trigger("reset");
+});
+// Delete button actions on click event
+$("#hero-list").on("click", ".btn-danger", removeHero());
```

Comment:
You try to pass removeHero as an argument here, but call it instead... this is the difference between `getTwo` (a function) and `getTwo()` (a number). You want to pass the function, removeHero, as the argument, because that's the code you want executed after the button is clicked.